### PR TITLE
Task 2

### PR DESCRIPTION
**Pull request by YuriiBobeshko**

### CODE REVIEW

File: src/webSocket/getAutoSuggestUsers.ts [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+
+import { server } from '../index';
+import { filterUserByLogin } from '../controllers/user';
+
+export const getAutoSuggestUsers = () => {
+  const wss = new WebSocket.Server({ server });
+
+  wss.on('connection', (ws: WebSocket) => {
+    ws.on('message', (message: string) => {
+      const { text, limit = 5 } = JSON.parse(message);
```
Comment (boale) [inline src/webSocket/getAutoSuggestUsers.ts]: wrap with try/catch

File: src/webSocket/getAutoSuggestUsers.ts [hunk 2]
```diff
@@ -0,0 +1,16 @@
+import WebSocket from 'ws';
+
+import { server } from '../index';
+import { filterUserByLogin } from '../controllers/user';
+
+export const getAutoSuggestUsers = () => {
```
Comment (boale) [inline src/webSocket/getAutoSuggestUsers.ts]: server can be passed as an argument

File: src/webSocket/getAutoSuggestUsers.ts [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+  const wss = new WebSocket.Server({ server });
+
+  wss.on('connection', (ws: WebSocket) => {
+    ws.on('message', (message: string) => {
```
Comment (boale) [inline src/webSocket/getAutoSuggestUsers.ts]: Callbacks can be extracted as separate functions

File: src/routes/user.ts [hunk 4]
```diff
@@ -0,0 +1,41 @@
+import validatorJoi from 'express-joi-validation';
+import express from 'express';
+
+import { IGetUserAuthInfoRequest } from '../types/user';
+import { createUser, deleteUser, getUser, upDateUser } from '../controllers/user';
+import { createUserSchema, deleteUserSchema, getUserByIdSchema, upDateUserSchema } from '../schemas/user';
+
+const Router = express.Router();
```
Comment (boale) [inline src/routes/user.ts]: please, start instances with lower case.

File: src/routes/user.ts [hunk 5]
```diff
@@ -0,0 +1,41 @@
...
+const validator = validatorJoi.createValidator();
+
+Router.param('id', (req: IGetUserAuthInfoRequest, res, next) => {
```
Comment (boale) [inline src/routes/user.ts]: 👍

File: src/webSocket/getAutoSuggestUsers.ts (hunk 3 above)
Comment (YuriiBobeshko) [reply src/webSocket/getAutoSuggestUsers.ts]: fixed

File: src/routes/user.ts (hunk 4 above)
Comment (YuriiBobeshko) [reply src/routes/user.ts]: fixed

File: src/routes/user.ts (hunk 5 above)
Comment (YuriiBobeshko) [reply src/routes/user.ts]: fixed

File: src/webSocket/getAutoSuggestUsers.ts (hunk 2 above)
Comment (YuriiBobeshko) [reply src/webSocket/getAutoSuggestUsers.ts]: fixed

File: src/webSocket/getAutoSuggestUsers.ts (hunk 1 above)
Comment (YuriiBobeshko) [reply src/webSocket/getAutoSuggestUsers.ts]: fixed

**Review by boale — APPROVED** [review]

**Pull request closed without merging**